apps/authentication-service/cypress/e2e/rateLimiting.cy.ts

```diff
@@ -1,4 +1,4 @@
-const testLimit: number = 30;
+const testLimit: number = 40;
```

Comment:
Increased this to accommodate additional e2e tests. Is there a better way to do this? Since tests can happen in parallel, they all add to the rate limit max.